bindings/python/src/session.cpp

```diff
@@ -748,6 +748,7 @@ void bind_session()
         )
 #endif
         .def("post_torrent_updates", allow_threads(&lt::session::post_torrent_updates))
+        .def("post_session_stats", allow_threads(&lt::session::post_session_stats))
```

Comment:
would you mind adding a call to this function in test.py as well, just as a smoke test?